frontend/src/game/board/builder.js

```diff
@@ -40,6 +42,12 @@ module.exports = {
         newCard.classList.remove("col-1");
         newCard.classList.add("col-2");
         newCard.innerHTML = cardData["power"];
+        newCard.addEventListener('click', function() {
+            if (status.clickedCard !== undefined &&
```

Comment:
`if (status.clickedCard !== undefined)   `
is equivalent to 
` if (status.clickedCard)`

Reply:
Thanks.